pkg/sac/scope_key.go

```diff
@@ -159,7 +160,8 @@ type ScopeSuffix []ScopeKey
 
 // TryAllowed implements the ScopePredicate interface.
 func (i ScopeSuffix) TryAllowed(sc ScopeChecker) bool {
```

Comment:
Would it make sense to rename the `ScopePredicate` `TryAllowed` function to `Allowed` or something similar as the two-pass logic `TryAllowed` / `PerformChecks` / `TryAllowed` is not needed anymore ?
This is candidate for a follow-up pull request if not already planned.

Reply:
It's done here https://github.com/stackrox/stackrox/pull/2779